source/patterns/@aws-solutions-constructs/aws-fargate-s3/lib/index.ts

```diff
@@ -208,20 +209,24 @@ export class FargateToS3 extends Construct {
       if (props.bucketPermissions.includes('Delete')) {
         bucket.grantDelete(this.service.taskDefinition.taskRole);
       }
-      if (props.bucketPermissions.includes('Put')) {
-        bucket.grantPut(this.service.taskDefinition.taskRole);
-      }
       if (props.bucketPermissions.includes('Read')) {
         bucket.grantRead(this.service.taskDefinition.taskRole);
       }
-      if (props.bucketPermissions.includes('ReadWrite')) {
```

Comment:
We can't delete this functionality as that will break existing clients - we should drop them from the doc but leave the code that supports them in place along with comments that 'Put' and 'ReadWrite' are deprecated but this code remains for backwards compatibility.

```
   // ReadWrite and Put are deprecated, but included here for backwards compatibility
   if (props.bucketPermissions.includes('Write') || 
        props.bucketPermissions.includes{'ReadWrite') ||
        props.bucketPermissions.includes('Put') )
   {
        ...
   }
   if (props.bucketPermissions.includes('Read') || props.bucketPermissions.include('ReadWrite') {
      ...
   }

etc.

  if (props.bucketPermissions.includes('ReadWrite') || props.bucketPermissions.includes('Put')) {
    printWarning('Put and ReadWrite are deprecated, please switch to some combination of Read, Write and Delete');
  }

```